src/locales/en.json

```diff
@@ -1,6 +1,10 @@
 {
+  "Acquisition": {
+    "success": "Successful acquisition"
```

Comment:
I think this is correct but I'm more used to the `Acquisition success` form in comparison to classic `download complete` or `upload success` messages.

But I'm not sure, I expected to find more references on internet but maybe i'm using bad search keywords